mineral-ui/mineral-ui
# [Button] Update styles

### Description

- Update Button styles to match new spec, mostly affects padding & application of color
- Remove no-longer-needed Button styles (yay)
- Remove RTL styles to fix #212 
- Update mineralTheme with new color applications
- Add 'jumbo' size to Button
- Update Icon sizes to match what is used in new Button spec
- Update Button examples
- Update snapshots
- Add `pxToEm` helper to component-utils and apply to styleVariables

Closes #195; closes #179; closes #212

### Motivation and context

Better Button! Seriously, just look ⬇️ .

### Screenshots or videos, if appropriate

![screen shot 2017-07-27 at 2 31 04 pm](https://user-images.githubusercontent.com/486540/28691008-99a070bc-72d8-11e7-8641-ac94bea375fa.png)
![screen shot 2017-07-27 at 2 32 35 pm](https://user-images.githubusercontent.com/486540/28691013-9bbf2ec4-72d8-11e7-98d1-d8d2aab871fa.png)

### How to test
1. `cd mineral-ui/packages/button && npm start`
1. See the new styles, use Dev Tools to confirm they match spec

### Types of changes

- Breaking change (fix or feature that would cause existing functionality to change)

The theme changed. Button also no longer has some previously-available theme variables.

### Checklist




### How does this PR make you feel?

![Push the button; do something cool](https://media.giphy.com/media/l2JdSlA1a1zKVAyze/giphy.gif)
